src/components/vr_cooperation/src/vr_module.cc

```diff
@@ -0,0 +1,104 @@
+/*
+ Copyright (c) 2013, Ford Motor Company
+ All rights reserved.
+
+ Redistribution and use in source and binary forms, with or without
+ modification, are permitted provided that the following conditions are met:
+
+ Redistributions of source code must retain the above copyright notice, this
+ list of conditions and the following disclaimer.
+
+ Redistributions in binary form must reproduce the above copyright notice,
+ this list of conditions and the following
+ disclaimer in the documentation and/or other materials provided with the
+ distribution.
+
+ Neither the name of the Ford Motor Company nor the names of its contributors
+ may be used to endorse or promote products derived from this software
+ without specific prior written permission.
+
+ THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+ ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+ LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+ CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+ SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+ INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+ CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ POSSIBILITY OF SUCH DAMAGE.
+ */
+
+#include "vr_cooperation/vr_module.h"
+#include "utils/logger.h"
+
+namespace vr_cooperation {
+
+using functional_modules::ProcessResult;
+
+CREATE_LOGGERPTR_GLOBAL(logger_, "VRModule")
+
+VRModule::VRModule()
+  : GenericModule(kVRModuleID) {
+}
+
+VRModule::~VRModule() {
+}
+
+functional_modules::PluginInfo VRModule::GetPluginInfo() const {
+  LOG4CXX_AUTO_TRACE(logger_);
+  return plugin_info_;
+}
+
+ProcessResult VRModule::ProcessMessage(application_manager::MessagePtr msg) {
+  LOG4CXX_AUTO_TRACE(logger_);
+  return ProcessResult::PROCESSED;
+}
+
+ProcessResult VRModule::ProcessHMIMessage(
+  application_manager::MessagePtr msg) {
+  LOG4CXX_AUTO_TRACE(logger_);
+  return ProcessResult::PROCESSED;
+}
+
+void VRModule::RemoveAppExtension(uint32_t app_id) {
+  LOG4CXX_AUTO_TRACE(logger_);
```

Comment:
please add a comment that for VR Service it is not implemented since there is no app extension functionality in VR Service